Packs/FeedRecordedFuture/Integrations/FeedRecordedFuture/FeedRecordedFuture_test.py

```diff
@@ -1,7 +1,7 @@
 import pytest
 from collections import OrderedDict
-from FeedRecordedFuture import get_indicator_type, get_indicators_command, Client, fetch_indicators_command
```

Comment:
why did you move the imports into the test funcs?

Reply:
This changed during the writing of the new tests I added. I changed it back.